tests/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PCAPS_DIR=tests/pcaps
+[ "$(ls -A --ignore='.??*' tests/pcaps)" ] && echo "Skipping downloading PCAPs" || gsutil -m cp gs://static.up9.io/mizu/test-pcap/\*.pcap $PCAPS_DIR
```

Comment:
Getting the following error when running:

`ls: unrecognized option `--ignore=.??*'`

Reply:
Additional error:
`tests/run.sh: line 19: timeout: command not found
Traceback (most recent call last):
  File "/Users/igorgov/dev/mizu/tests/test.py", line 17, in <module>
    import websocket
ModuleNotFoundError: No module named 'websocket'
tests/run.sh: line 27: kill: (-64291) - No such process
make: *** [test-run] Error 1`

Reply:
should we have a requirements.json, and part of the script create venv and install the requirements into it?